rsocket/internal/RSocketConnectionManager.cpp

```diff
@@ -37,9 +36,9 @@ RSocketConnectionManager::~RSocketConnectionManager() {
     map.swap(*locked);
   }
 
-  for (auto& kv : map) {
+  for (auto &kv : map) {
```

Comment:
Why `auto &kv` (and all following uses of `&` later in this file)?  The rest of the codebase does `auto& kv`.

Reply:
I am using the clang+format, auto formatter. I have noticed this too but I wonder if we should use clang+format or I should just revert this line?

Reply:
We do use `clang-format` and it doesn't format things like this for any of us...  Is your IDE respecting the `.clang-format` file we have in our top-level directly?

Reply:
Ops, you are right. I was unaware of this file 💯